lib/std/child_process.zig

```diff
@@ -57,6 +57,12 @@ pub const ChildProcess = struct {
 
     expand_arg0: Arg0Expand,
 
+    /// Use this for additional posix spawn attributes.
```

Comment:
These are the other relevant bits to know for users.

Reply:
removed with different api: either user takes ownership and must deinit or (default) its within the macos posix_spawn function.